sdk/core/Azure.Core/api/Azure.Core.net6.0.cs

```diff
@@ -1145,6 +1136,16 @@ public abstract partial class ObjectSerializer
         public abstract System.Threading.Tasks.ValueTask SerializeAsync(System.IO.Stream stream, object? value, System.Type inputType, System.Threading.CancellationToken cancellationToken);
         public virtual System.Threading.Tasks.ValueTask<System.BinaryData> SerializeAsync(object? value, System.Type? inputType = null, System.Threading.CancellationToken cancellationToken = default(System.Threading.CancellationToken)) { throw null; }
     }
+    public enum PropertyNamingConvention
+    {
+        None = 0,
+        CamelCase = 1,
+    }
+    public partial class ProtocolOptions
```

Comment:
Is this name too general for what it does?

Reply:
How would you feel about ProtocolMethodOptions instead?